app/components/Modal.tsx

```diff
@@ -254,7 +254,7 @@ const Header = styled(Flex)`
 const Small = styled.div`
   animation: ${fadeAndScaleIn} 250ms ease;
 
-  margin: auto auto;
+  margin: 25vh auto auto auto;
```

Comment:
Note that this affects dialogs across the board.

Reply:
I believe we need to define a max height too, maybe 70vh?

<img width="697" alt="image" src="https://github.com/outline/outline/assets/380914/9e10506e-51cd-45fe-aba9-85d31205dd83">

Reply:
Just a note on this - setting max height leads us to manage overflow which provokes to manage scroll 😅 

Looked into all this yesterday, almost there...

Reply:
🤞 


https://github.com/outline/outline/assets/15694566/68b03146-af78-4dc5-a3a6-992146a28710